### PR TITLE
create sustainability datasets page

### PR DESCRIPTION
**Pull request by n-h-diaz**

### CODE REVIEW

File: datasets/united_states.md [hunk 1]
```diff
@@ -138,9 +99,3 @@ Data Commons has imported the potential for solar energy generation in different
 The [Census of Agriculture](https://www.nass.usda.gov/AgCensus/) contains information about land use, operators, production, and finances of US farms and ranches. Data Commons contains several of the Census tables relating to crop production and operator demographics from the 2017 Census.
```
Comment (pradh) [inline datasets/united_states.md:99]: should we move agriculture too?

Comment (n-h-diaz) [reply datasets/united_states.md]: done

**Review by pradh — APPROVED** [review]
Thanks!

**Review by beets — COMMENTED** [review]
thanks, this looks good!

**Pull request merged**